pkgs/by-name/do/dosbox-staging/package.nix

```diff
@@ -32,67 +32,15 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "dosbox-staging";
-  version = "0.80.1";
+  version = "0.81.0";
 
   src = fetchFromGitHub {
     owner = "dosbox-staging";
     repo = "dosbox-staging";
     rev = "v${finalAttrs.version}";
-    hash = "sha256-I90poBeLSq1c8PXyjrx7/UcbfqFNnnNiXfJdWhLPGMc=";
+    hash = "sha256-NP/p/eHbRjfdMwQBC9OAL01pmlHDiYq2OO2KgFWHero=";
   };
 
-  patches = [
-    # Pull missind SDL2_net dependency:
-    #   https://github.com/dosbox-staging/dosbox-staging/pull/2358
-    (fetchpatch {
```

Comment:
Remove `fetchpatch` from the input set too.

Reply:
Done. Thanks.